### PR TITLE
Documentation Update: Feature Selection Profiles, Data Splitting, and Trained Models

### PR DESCRIPTION
**Pull request by axiomcura**
This PR adds documentation on the different feature selection profiles generated, the data splitting approach, and the models trained in this study

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,15 +133,28 @@ All holdout datasets were removed from the main dataset.
 The remaining data was then split into training and testing sets using an 80/20 train/test split, respectively.
 We used the `stratify` parameter to preserve the distribution of labels, preventing label imbalances that could lead to classification problems.
 
+Here’s an improved version of your paragraph for clarity and conciseness:
+
+Here’s a clearer and more concise version of your text:
```
Comment (jenna-tomkinson) [inline README.md]: Recommend removing this part as it does not relate to the text 😉

File: README.md [hunk 2]
```diff
@@ -111,6 +111,12 @@ We identified the features shared between the `cell-injury` dataset and the JUMP
 We then applied feature selection using [Pycytominer](https://github.com/cytomining/pycytominer) to extract the most informative and least redundant features.
 This process produced our `feature-selected` profiles, which were subsequently used to train the multi-class logistic regression model.
 
+As a result, we generated two distinct types of feature-selected profiles:
+
+- **JUMP-Aligned Feature-Selected Cell Injury Profiles:** These profiles were created by applying feature selection to the cell injury profiles were the feature space is aligned with the JUMP data.
```
Comment (jenna-tomkinson) [inline README.md]:
```suggestion
- **JUMP-Aligned Feature-Selected Cell Injury Profiles:** These profiles were created by applying feature selection to the cell injury profiles where the feature space is aligned with the JUMP data.
```

File: README.md [hunk 3]
```diff
@@ -111,6 +111,12 @@ We identified the features shared between the `cell-injury` dataset and the JUMP
...
+
+- **Feature-Selected Cell Injury Profiles:** These profiles were generated by applying feature selection exclusively to the cell injury dataset, without any alignment to the JUMP data.
```
Comment (jenna-tomkinson) [inline README.md]:
```suggestion
- **Feature-Selected Cell Injury Profiles:** These profiles were generated by applying feature selection exclusively to the cell injury dataset, without any alignment to the JUMP data feature space.
```

File: README.md [hunk 4]
```diff
@@ -127,15 +133,28 @@ All holdout datasets were removed from the main dataset.
...
+
 ### Training the model
 
-We trained a multi-class logistic regression model using randomized cross-validation for hyperparameter tuning to fine-tune our model.
-Our logistic regression model was set to `multi_class="multinomial"`, indicating that it applies a softmax approach to handle and classify multiple classes.
-The model also included `class_weight="balanced"` to ensure the model pays more attention to minority classes during training, preventing label imbalance or bias.
+In this study, we developed two distinct types of multi-class regression models:
+
+1. **Model Trained on Feature-Selected Cell Injury Profiles**
+2. **Model Trained on Aligned Feature-Selected Cell Injury Profiles**
```
Comment (jenna-tomkinson) [inline README.md]:
```suggestion
2. **Model Trained on JUMP-Aligned Feature-Selected Cell Injury Profiles**
```

File: README.md [hunk 5]
```diff
@@ -127,15 +133,28 @@ All holdout datasets were removed from the main dataset.
...
+
+We trained our models using randomized cross-validation for hyperparameter tuning to fine-tune our model.
+Our logistic regression models was set to `multi_class="multinomial"`, indicating that it applies a softmax approach to handle and classify multiple classes.
+The models also included `class_weight="balanced"` to ensure the model pays more attention to minority classes during training, preventing label imbalance or bias.
 
 ### Applying model to JUMP data
 
-We applied our trained multi-class logistic regression model to the JUMP dataset to predict injuries present in each well.
+We applied our trained shuffled and non shuffled aligned feature selected multi-class logistic regression model to the JUMP dataset to predict injuries present in each well.
```
Comment (jenna-tomkinson) [inline README.md]:
```suggestion
We applied our trained shuffled and non-shuffled aligned feature-selected multi-class logistic regression model to the JUMP dataset to predict injuries present in each well.
```

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! I left a few comments to address, nice job!

**Comment by axiomcura** [comment]
Thanks now merging!

**Pull request merged**